run_tests.sh

```diff
@@ -97,6 +101,13 @@ export HOME="/root"
 nitro-cli build-enclave --docker-uri 667861386598.dkr.ecr.us-east-1.amazonaws.com/enclaves-samples:vsock-sample \
 	--output-file test_images/vsock-sample.eif || test_failed
 
+# Build enclave image using Docker ENTRYPOINT instruction
+sed -i 's/CMD/ENTRYPOINT/g' examples/"${ARCH}"/hello/Dockerfile
```

Comment:
can we create a copy of the directory and not modify in place ?

Reply:
I can do that in the next revision.